src/headset/oculus_mobile.c

```diff
@@ -0,0 +1,231 @@
+#include "lib/glfw.h"
+#include "headset/headset.h"
+#include "BridgeLovr.h"
+#include "math/quat.h"
+#include "graphics/graphics.h"
+#include "math/mat4.h"
+#include "lib/glad/glad.h"
+
+static void (*renderCallback)(void*);
+static void* renderUserdata;
+
+static float offset;
+
+void lovrOculusMobileDraw(int framebuffer, int width, int height, float *eyeViewMatrix, float *projectionMatrix) {
+  lovrGpuDirtyTexture();
+
+  CanvasFlags flags = {0};
+  Canvas *canvas = lovrCanvasCreateFromHandle(width, height, flags, framebuffer, 0, 0, 1, true);
+
+  Camera camera = { .canvas = canvas, .stereo = false };
+  memcpy(camera.viewMatrix[0], eyeViewMatrix, sizeof(camera.viewMatrix[0]));
+  mat4_translate(camera.viewMatrix[0], 0, -offset, 0);
+
+  memcpy(camera.projection[0], projectionMatrix, sizeof(camera.projection[0]));
+
+  lovrGraphicsSetCamera(&camera, true);
+
+  renderCallback(renderUserdata);
+
+  lovrGraphicsSetCamera(NULL, false);
+  lovrRelease(canvas);
+}
+
+// TODO
+void lovrHeadsetUpdate(float dt) {
+}
```

Comment:
This isn't doing anything right?

Reply:
Yup, accidental leftover from an earlier version of the code. Will remove